src/selector/index.js

```diff
@@ -1,47 +1,50 @@
-import { builder } from '../builder';
+import { builder } from "../builder";
 
-function extractProps () {
```

Comment:
Retain the space before parens, this is part of the semistandard specs. Apply to all.